Content.Server/Objectives/ObjectivesSystem.cs

```diff
@@ -134,6 +134,9 @@ private void AddSummary(ref string result, string agent, List<EntityUid> minds)
 
             foreach (var objectiveGroup in objectives.GroupBy(o => Comp<ObjectiveComponent>(o).Issuer))
             {
+                //TO DO:
+                //check for the right group here. Getting the target issuer is easy: objectiveGroup.Key
```

Comment:
This is terrible, I have spent so much time trying to fix this problem. Maybe someone will figure it out quickly, I don't know.
![image](https://github.com/space-wizards/space-station-14/assets/96445749/f7e10bf8-daeb-43ec-8b51-7d1a5f31d9dd)